docs/core/install/linux-ubuntu.md

```diff
@@ -44,6 +45,12 @@ The following versions of .NET Core are no longer supported. The downloads for t
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
 
+## 20.10 ✔️
+
+.NET 5 and .NET Core 3.1 package feeds for Ubuntu 20.10 currently have an issue. For more information about the issue, see [GitHub](https://github.com/dotnet/core/issues/5549). This article will be updated when the issue is resolved.
```

Comment:
It affects all packages, 2.1 as well. 😢 The "simple" workaround only works for 5.0 and 3.1 because we released an update for 5.0/3.1 that works on 20.10. You'd have to use both workarounds to get 2.1 working because it is suffering from both https://github.com/dotnet/core/issues/5549 and https://github.com/dotnet/core/issues/5352.